### ISSUE TITLE
Request to Telldus sensor list API failed. Reason. “Certificate has expired”

### ISSUE THREAD

**Issue by uffe6** [johnlemonse/homebridge-telldus#80]
Homebridge has stopped working all of a sudden. Trying to debug (se attached image) I’m getting that the error is that it fails to request the the Telldus sensor list. Under Telldus API. Reason: certificate expired. 

Does anybody know what’s wrong? 

![F4313C35-D6A5-4CE2-B942-6C1055AF2CC7](https://user-images.githubusercontent.com/66196908/83346486-0c105f00-a31d-11ea-8df6-bf12ea2bc6b0.png)

**Comment by mifi** [comment]
Is your system date and time correct?

**Comment by uffe6** [comment]
Hi! no it’s not. Date yes. Time Is 2 hours off. I’m in CET. System is at CET -2 hours. Can that be the reason?

**Comment by mifi** [comment]
Can you try curl -vvv https://api.telldus.com/

**Comment by uffe6** [comment]
![61109BB5-CF38-4727-B0E2-1F77A34AE969](https://user-images.githubusercontent.com/66196908/83348080-c9a14f00-a329-11ea-9720-028aaa731f8a.png)
![0282BC94-D388-4E0B-96F7-A9DEA8AB8EA0](https://user-images.githubusercontent.com/66196908/83348081-cad27c00-a329-11ea-9a31-139dad5225c7.png)

**Comment by uffe6** [comment]
What does it mean that the SSL certificate has expired?

**Comment by mifi** [comment]
Are you running newest version of homebridge-telldus package?

**Comment by uffe6** [comment]
Version 1.0.7

Is that the newest?

**Comment by mifi** [comment]
Yea. and nodejs version? try both 8 and 12. If that doesn't work then I really don't understand why it suddently stopped working, unless you did anything

**Comment by uffe6** [comment]
Ok. I did nothing. Just stopped working. The only thing I did was restarting my pi. And then restating homebridge. Then started to debug and found this....

**Comment by mifi** [comment]
Ok. Well I think the problem is at Telldus' side, because curl is also giving the same error. Maybe you can try to contact them and ask if they did any server changes.

**Comment by uffe6** [comment]
Yes. Already done that. Let’s see what they reply. Thanks for the help!

**Comment by mifi** [comment]
is this resolved now?

**Comment by uffe6** [comment]
The issue just went away. So it was “solved”

however the same problem occurred again…just yesterday.
